src/main/oshelper.cpp

```diff
@@ -159,6 +160,26 @@ QList<QString> OSHelper::grabQrCodesFromScreen() const
     return codes;
 }
 
+QList<QString> OSHelper::grabQrCodesFromClipboard() const
+{
+    QList<QString> codes;
+
+    try
+    {
+        QClipboard *clipboard = QGuiApplication::clipboard();
+        const QImage clipboardImage = clipboard->image();
```

Comment:
```suggestion
        const QImage clipboardImage = clipboard->image();
        if (clipboardImage.isNull()) {
            return codes;
        }
```

Untested but this should be added.